tests/guid.spec.ts

```diff
@@ -2,57 +2,33 @@ import { assert, expect } from "chai";
 import "mocha";
 import { Guid } from "../lib/guid";
 
-describe("Guid test", () => {
-    
-    it("Should create a guid", () => {
+describe("Guid", () => {    
+    const exampleGuid = "0315642c-a069-9f3e-1852-9adf2d075b93";
+
+    it("Should create & validate a random guid", () => {
         const wrong = "wrongguid";
         expect(Guid.isGuid(wrong)).equal(false);
 
         const right = Guid.create();
         expect(Guid.isGuid(right)).equal(true);
     });
 
-    it("Should raw a guid", () => {
+    it("Should parse & validate a guid", () => {
         const wrong = "wrongguid";
         expect(Guid.isGuid(wrong)).equal(false);
 
-        const right = Guid.raw();
-        expect(Guid.isGuid(right)).equal(true);
+        expect(Guid.isGuid(exampleGuid)).equal(true);
     });
 
-    it("Should compare another guid", () => {
+    it("Should compare to another guid", () => {
         const wrong = Guid.create();
         expect(wrong.equals(Guid.create())).equal(false);
         
         const right = Guid.create();
         expect(right.equals(right)).equal(true);
```

Comment:
```suggestion
        expect(right.equals(right)).equal(true);
```
doesn't make sense at all, please compare something different! :)

Reply:
You're right, I reqrote these comparison tests.